<ui version="4.0" >
 <class>latLonCoordinateWidget</class>
 <widget class="QWidget" name="latLonCoordinateWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>104</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="1" >
    <widget class="QLineEdit" name="latDegLineEdit" />
   </item>
   <item row="0" column="2" >
    <widget class="QLabel" name="label_7" >
     <property name="text" >
      <string>deg</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" >
    <widget class="QLineEdit" name="latMinLineEdit" />
   </item>
   <item row="0" column="4" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>min</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5" >
    <widget class="QLineEdit" name="latSecLineEdit" />
   </item>
   <item row="0" column="6" >
    <widget class="QLabel" name="label_9" >
     <property name="text" >
      <string>sec</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="minimumSize" >
      <size>
       <width>71</width>
       <height>31</height>
      </size>
     </property>
     <property name="title" >
      <string>N/S</string>
     </property>
     <widget class="QWidget" name="layoutWidget_1" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>10</y>
        <width>71</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <item>
        <widget class="QRadioButton" name="latNRadioButton" >
         <property name="text" >
          <string>N</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="latSRadioButton" >
         <property name="text" >
          <string>S</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8pt;">&lt;span style=" font-size:9pt;">Longitude:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="lonDegLineEdit" />
   </item>
   <item row="1" column="2" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>deg</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QLineEdit" name="lonMinLineEdit" />
   </item>
   <item row="1" column="4" >
    <widget class="QLabel" name="label_8" >
     <property name="text" >
      <string>min</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5" >
    <widget class="QLineEdit" name="lonSecLineEdit" />
   </item>
   <item row="1" column="6" >
    <widget class="QLabel" name="label_10" >
     <property name="text" >
      <string>sec</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="minimumSize" >
      <size>
       <width>71</width>
       <height>31</height>
      </size>
     </property>
     <property name="title" >
      <string>E/W</string>
     </property>
     <widget class="QWidget" name="layoutWidget_2" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>10</y>
        <width>73</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3" >
       <item>
        <widget class="QRadioButton" name="lonWRadioButton" >
         <property name="text" >
          <string>W</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="lonERadioButton" >
         <property name="text" >
          <string>E</string>
         </property>
         <property name="checked" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8pt;">&lt;span style=" font-size:9pt;">Latitude&lt;/span>&lt;span style=" font-size:9pt; font-weight:600;">:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>layoutWidget</zorder>
  <zorder>label_3</zorder>
 </widget>
 <tabstops>
  <tabstop>latDegLineEdit</tabstop>
  <tabstop>latMinLineEdit</tabstop>
  <tabstop>latSecLineEdit</tabstop>
  <tabstop>latNRadioButton</tabstop>
  <tabstop>latSRadioButton</tabstop>
  <tabstop>lonDegLineEdit</tabstop>
  <tabstop>lonMinLineEdit</tabstop>
  <tabstop>lonSecLineEdit</tabstop>
  <tabstop>lonWRadioButton</tabstop>
  <tabstop>lonERadioButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
